runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerBarOverlay.java

```diff
@@ -60,6 +61,10 @@
 	private final PrayerPlugin plugin;
 
 	private boolean showingPrayerBar;
+	private boolean debounce;
```

Comment:
Rather than adding this field, add a subscriber to the plugin for MenuOptionClicked, check that the prayer orb is being clicked on, then call some function in this overlay which indicates it has been clicked.